assignment/index.html

```diff
@@ -1,25 +1,38 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <!-- Change the <title> from "Page Title" to "Hello World" -->
-    <title>Unit 1 Assignment</title>
+    <title>Hello World</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    <script type="text/javascript"
+        src="/Users/hannahengelhart/Documents/Prime/assignment-unit-1/assignment/helloWorld.js"></script>
```

Comment:
Whoops! This is a path relative to the root directory on your computer. While this would work in your Terminal, this won't work in the browser. The path must be relative to the `index.html` file itself, or a URL starting with `https://` or `http://`. So in this case, `src="helloWorld.js"` would work, but this will this code will throw an error.